ops/dockerfiles/server.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM mycro/solc-python:latest
+
+#setup uwsgi used in prod environments
+RUN apt-get update
+RUN pip3 install --upgrade pip
+RUN pip3 uninstall uwsgi
+RUN apt-get -y install build-essential python-dev
```

Comment:
sanity check on whether this needs to be `python3-dev`

Reply:
according to the docs on uwsgi I read we need python-dev for uwsgi to build correctly